capstone/client/src/components/StoresPage.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Component } from 'react';
+import { Grid, Card, Typography, List, ListItem, ListItemText, Button, ExpansionPanelActions } from '@material-ui/core';
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { Map, GoogleApiWrapper } from 'google-maps-react';
+
+import { Store } from './Store';
+
+const stores = [{
+    name: 'Safeway',
+    address: '11050 Bollinger Canyon Rd, San Ramon',
+    totalItemsFound: 5,
+    items: [{name: 'O Organics Organic Whole Milk with Vitamin D - 1 Gallon', price: 4.55}, 
+      {name: 'Lucerne Milk Reduced Fat 2% Milkfat 1 Gallon - 128 Fl. Oz.', price: 2.55},
+      {name: 'Silk Almondmilk Original Unsweetened - Half Gallon', price: 4.33}],
+    distance: 3.1,
+    price: 8.96
+}, 
+{
+    name: 'Walmart',
+    address: '9100 Alcosta Blvd, San Ramon',
+    totalItemsFound: 2,
+    items: [{name: 'O Organics Organic Whole Milk with Vitamin D - 1 Gallon', price: 4.55}, 
+      {name: 'Lucerne Milk Reduced Fat 2% Milkfat 1 Gallon - 128 Fl. Oz.', price: 2.55},
+      {name: 'Silk Almondmilk Original Unsweetened - Half Gallon', price: 1.44}],
+    distance: 2.3,
+    price: 16.45,
+}
+]
+
+class StorePage extends Component { 
+    constructor(props) {
+      super(props)
+    }
+
+    render() {
+
+      const storeOverviewCards = stores.map((store) => (
+          <div>
```

Comment:
i think this file is going to get really big like the ListPage too. So maybe it's cleaner to move this to a new file called `StoreOverviewCard`  and another component called `MapCard` that contains all the styling. Then this file can just handle most of the interaction logic, not so much the styling.